readthedocs/profiles/views.py

```diff
@@ -169,3 +176,81 @@ class TokenDeleteView(TokenMixin, DeleteView):
 
     def get_object(self, queryset=None):  # noqa
         return self.request.user.auth_token
+
+
+class UserSecurityLogView(PrivateViewMixin, ListView):
+    model = AuditLog
+    template_name = 'profiles/private/security_log.html'
+    days_limit = settings.RTD_DEFAULT_LOGS_RETENTION_DAYS
+
+    def get(self, request, *args, **kwargs):
+        download_data = request.GET.get('download', False)
+        if download_data:
+            return self._get_csv_data()
+        return super().get(request, *args, **kwargs)
+
+    def _get_csv_data(self):
+        headers = [
+            'Date',
+            'User',
+            'Project',
+            'Organization',
+            'Action',
+            'IP',
+            'Browser',
+        ]
+        data = (
+            self._get_queryset()
+            .values_list(
+                'created',
+                'log_user_username',
+                'log_project_slug',
+                'log_organization_slug',
+                'action',
+                'ip',
+                'browser',
+            )
```

Comment:
Seems like we want to match the user-shown value & the queryset value, probably in a dict (`{'Date': 'created'}`)? They are disconnected here, and that will likely lead to bugs.